tcmu-runner/build/build_rpm

```diff
@@ -86,9 +86,10 @@ echo "\"\"\"" >> tcmu-runner.cfg
 cat tcmu-runner.cfg
 
 ## Build the binaries with mock
+## disable 'glfs' because centos9 packages aren't available
 echo "Building RPMs"
-sudo mock --verbose -r tcmu-runner.cfg --scrub=all
-sudo mock --verbose -r tcmu-runner.cfg --resultdir=$WORKSPACE/dist/RPMS/ ${SRPM} || ( tail -n +1 $WORKSPACE/dist/RPMS/{root,build}.log && exit 1 )
+sudo mock --verbose --without glfs -r tcmu-runner.cfg --scrub=all
+sudo mock --verbose --without glfs -r tcmu-runner.cfg --resultdir=$WORKSPACE/dist/RPMS/ ${SRPM} || ( tail -n +1 $WORKSPACE/dist/RPMS/{root,build}.log && exit 1 )
```

Comment:
Maybe you should also disable `qcow,file,zbc` :

```
o- backstores .......................................................... [...]
  o- user:glfs .......................................... [Storage Objects: 0]
  o- user:qcow .......................................... [Storage Objects: 0]
  o- user:rbd ........................................... [Storage Objects: 0]
  o- user:file .......................................... [Storage Objects: 0]
  o- user:zbc ........................................... [Storage Objects: 0]
```

Reply:
Maybe, but, those still build, and might be useful for some user(s).

Reply:
i had pushed a commit to disable other backends too, but hadn't seen @dmick's comment; i agree and reverted back to the version that Dan already tested